src/Visitors/RemoveArrayPropertyItem.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace RonasIT\Larabuilder\Visitors;
+
+use Illuminate\Support\Arr;
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Scalar;
+use PhpParser\Node\Stmt\Property;
+use RonasIT\Larabuilder\Exceptions\UnexpectedPropertyTypeException;
+
+class RemoveArrayPropertyItem extends SetPropertyValue
```

Comment:
it would be better to implement new abstract class `PropertyVisitor` class and use it as parent of this one and `SetPropertyValue` visitor